src/components/CtypeDetailView/CtypeDetailView.tsx

```diff
@@ -41,7 +41,7 @@ class CtypeDetailView extends React.Component<Props, State> {
             <div className="attributes">
               <div>
                 <label>Title</label>
-                <div>{cType.cType.metadata.title.default}</div>
+                <div>{cType.cType.schema.$id}</div>
```

Comment:
see comment above

Reply:
since `ctype` is of type `CTypeMetadata`, we should be able to get the title from the metadata

Reply:
This is done as of writing this comment